cdap-data-fabric-tests/src/test/java/co/cask/cdap/data/stream/LocalStreamDataFileTest.java

```diff
@@ -46,8 +49,13 @@ public static void init() throws IOException {
       new ConfigModule(cConf),
       new LocationRuntimeModule().getInMemoryModules(),
       new DataFabricLevelDBModule(),
-      new TransactionMetricsModule()
-    );
+      new TransactionMetricsModule(),
+      new AbstractModule() {
+        @Override
+        protected void configure() {
+          bind(StreamMetaStore.class).to(NoOpStreamMetaStore.class);
```

Comment:
Since you have it everywhere, can you just create a no op module class in the data-fabric-tests test src?